src/internet_identity/src/storage/storable_anchor.rs

```diff
@@ -0,0 +1,32 @@
+use crate::storage::anchor::Device;
+use candid::{CandidType, Deserialize};
+use ic_stable_structures::storable::Bound;
+use ic_stable_structures::Storable;
+use internet_identity_interface::internet_identity::types::MetadataEntry;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+#[derive(Clone, Debug, Default, CandidType, Deserialize, Eq, PartialEq)]
+pub struct StorableAnchor {
```

Comment:
I decided to postpone the renaming to the "identity" based naming. This will be done once we actually change the terminology in the business logic / migrate stable memory.